src/Data/Subscription/SubscriptionDataClass.ts

```diff
@@ -35,7 +35,10 @@ export const Subscription = provideDataClass('Subscription', {
         unfilteredSubscriptions,
       )
 
-      return { results: subscriptions, count: subscriptions.length }
+      return {
+        results: subscriptions.slice(0, params.limit()),
```

Comment:
* [ ] Oops! Checking why limit is not respected already I saw that `fetchSubscriptions` doesn't properly handle `continuation`. We should fix that and handle `limit` early. However, this also means that client-side filtering is not a good idea.